swablu/specific/reputation.py

```diff
@@ -195,8 +205,16 @@ async def process_cmd(message: Message):
 class GuildPointsHandler(tornado.web.RequestHandler):
     async def get(self, *args, **kwargs):
         try:
-            await self.render("points.html", title="SkyTemple - Guild Points",
-                              all_points=get_all_guild_points(), **DEFAULT_AUTHOR_DESCRIPTION)
+            if self.get_query_argument("json", None) is not None:
+                self.set_header('Content-Type', 'application/json')
+                all_credits = {}
+                for _, discord_id, __, points in get_all_guild_points():
+                    all_credits[discord_id] = points
```

Comment:
I'm confused, why is this named `all_credits`? Doesn't this code have to do with Guild Points?

Reply:
Whoops, yeah of course, got the terms confused while also working on the sprite server. Fixed.